.env.sample

```
@@ -1,7 +1,5 @@
-
-NEXT_PUBLIC_BASE_URL=<'server url here'>
-NEXT_PUBLIC_CHANNELTALK_PLUGIN_KEY=<'channelTalk plugin key here'>
-NEXT_PUBLIC_GA_ID=<'Google Analytics ID here'>
-NEXT_PUBLIC_SENTRY_AUTH_TOKEN=<'sentry auth token here'>
-NEXT_PUBLIC_SENTRY_DSN=<'sentry dsn here'>
\ No newline at end of file
+NEXT_PUBLIC_BASE_URL=http://localhost:8080/api
+NEXT_PUBLIC_CHANNELTALK_PLUGIN_KEY=READ_NOTION_PLEASE
+NEXT_PUBLIC_GA_ID=G-...
+NEXT_PUBLIC_SENTRY_AUTH_TOKEN=sntryu_...
+NEXT_PUBLIC_SENTRY_DSN=https://sentry.io/...
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)